OJP/OJP_Trips.xsd

```diff
@@ -228,26 +233,56 @@
 		</xs:annotation>
 		<xs:restriction base="xs:string">
-			<xs:enumeration value="fastest"/>
-			<xs:enumeration value="minChanges"/>
-			<xs:enumeration value="leastWalking"/>
-			<xs:enumeration value="leastCost"/>
+			<xs:enumeration value="fastest">
+				<xs:annotation>
+					<xs:documentation>Shortest duration somewhere in the future. This may present a shorter trip than the next earliest arrival (with latest departure).</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="minChanges">
+				<xs:annotation>
+					<xs:documentation>Minimise the number of interchanges as the first criterion.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="leastWalking">
+				<xs:annotation>
+					<xs:documentation>Shortest walking distance in meters, summed over all legs.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="leastCost">
+				<xs:annotation>
+					<xs:documentation>Cheapest fare, taking into account the applicable reductions. Might not base on actual cost, but an estimation.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
 			<xs:enumeration value="leastDistance">
 				<xs:annotation>
-					<xs:documentation>Least distance in meter. Mostly used for ALTERNATIVE MODE OF OPERATION</xs:documentation>
+					<xs:documentation>Least distance in metres. Mostly used for ALTERNATIVE MODE OF OPERATION and for ItModesToCover.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="earliestArrival">
+				<xs:annotation>
+					<xs:documentation>Earliest possible arrival time respecting the time constraints. (forward search)</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="latestDeparture">
+				<xs:annotation>
+					<xs:documentation>Latest departure time for a given arrival time. (backward search)</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="earliestArrivalAndLatestDeparture">
+				<xs:annotation>
+					<xs:documentation>Combines earliestArrival and latestDeparture, allowing to compress the departure time. (forward-backward-forward search)</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="levelEntrance"/>
+			<xs:enumeration value="levelEntranceOrBoardingAid"/>
+			<xs:enumeration value="noSingleStep">
+				<xs:annotation>
+					<xs:documentation>The user wants to minimise the number of single steps. Stairs or non-level entrances are not taken into account.</xs:documentation>
```

Comment:
```suggestion
					<xs:documentation>The user wants to avoid transfers without tactile guidance, as well as platforms and vehicles without auditory signals.</xs:documentation>
```

Reply:
PLUS *bestForAuditoryImpairment*